state/syncer/userAccountsSyncer.go

```diff
@@ -263,6 +266,11 @@ func (u *userAccountsSyncer) syncAccountDataTries(
 
 	wg.Wait()
 
+	err = common.ErrFromChan(leavesChannels.ErrChan)
+	if err != nil {
+		log.Error("error on getting all leaves", "err", err)
```

Comment:
Return the err?

Reply:
agree, critical error here: the nodes we've just written are missing.

Reply:
done